napari/tests/util.py

```diff
@@ -0,0 +1,89 @@
+import napari
+import numpy as np
+from napari.layers import (
+    Image,
+    Labels,
+    Points,
+    Shapes,
+    Vectors,
+    Surface
+)
+
+layer_test_data = [
+    (Image, np.random.random((10, 15)),  2),
+    (Image, np.random.random((10, 15, 20)), 3),
+    (Image, [np.random.random(s) for s in [(40, 20), (20, 10), (10, 5)]], 2),
+    (Labels, np.random.randint(20, size=(10, 15)), 2),
+    (Points, 20 * np.random.random((10, 2)), 2),
+    (Vectors, 20 * np.random.random((10, 2, 2)), 2),
+    (Shapes, 20 * np.random.random((10, 4, 2)), 2),
+    (Surface, (np.random.random((10, 3)), np.random.randint(10, size=(6, 3)),
+                 np.random.random(10)), 3)
+]
```

Comment:
can you add more ndim=3 tests where you can use the data from https://github.com/napari/napari/blob/6c5bb037e191bce3c05c5bf23f4f143ecc89a240/napari/tests/test_nD.py and then we can delete `test_nD.py` entirely

Reply:
for every layer type there should be both a 2D and 3D example, for Image there should also be a 4D example